.github/workflows/conformance.yaml

```diff
@@ -13,9 +13,8 @@ env:
 
 jobs:
   conform-kubernetes:
-    # Hosted on Equinix
-    # Docs: https://github.com/fluxcd/flux2/tree/main/.github/runners
-    runs-on: [self-hosted, Linux, ARM64, equinix]
+    runs-on:
+      group: "ARM64"
```

Comment:
what is the runner setup? How many runners do we have?

Reply:
This is custom group I created using GH EE, I used the default 50 max runners.